specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/databaseVulnerabilityAssessmentBaselines.json

```diff
@@ -61,15 +61,16 @@
           {
             "name": "baselineName",
             "in": "path",
-            "description": "The name of the vulnerability assessment rule baseline.",
+            "description": "The name of the vulnerability assessment rule baseline (default implies a baseline on a database level rule and master for server level rule).",
             "required": true,
             "type": "string",
             "enum": [
-              "default"
+              "Master",
+              "Default"
```

Comment:
I think this should be 'master' , 'default'. The C# code generator should capitalize the first letter for the enum name when it generated code.